src/integrationTest/groovy/wooga/gradle/unity/UnityTaskIntegrationSpec.groovy

```diff
@@ -355,8 +361,8 @@ abstract class UnityTaskIntegrationSpec<T extends UnityTask> extends UnityIntegr
         // "2019"       | false          | false       | false              | ""            | false
 
         "2018"       | true           | false       | true               | "#path"       | true
-        "2018"       | true           | true        | true               | ""            | true
-        "2018"       | false          | true        | false              | ""            | true
+        "2018"       | true           | true        | true               | null          | true
+        "2018"       | false          | true        | false              | null          | true
```

Comment:
Yes, since the function returns null, not an empty string.

Reply:
`resolveFilePath()`, the function has always return null by default or if no file was found.

The previous testing function had a different way to test whether it was a match (it checked `string.contains`, rather than the actual value of the operation). Details can be seen on the `MethodQueryTaskWriter`.